### ISSUE TITLE
Array types are unavailable?

### ISSUE THREAD

**Issue by MatanelGordon** [rustwasm/wasm-bindgen#3382]
### Summary

I was trying to play with `Uint32Array` as I found out it is unavailable.

is it intentional?

### Additional Details

<img width="398" alt="Screenshot 2023-04-06 at 23 52 40" src="https://user-images.githubusercontent.com/72258277/230491743-cd6f1c59-dd7c-473a-990a-a95c59c028cb.png">

**Comment by daxpedda** [comment]
I don't know what this screenshot is, but `Uint32Array` is available in the [`js-sys`](https://crates.io/crates/js-sys/0.3.61) crate: https://docs.rs/js-sys/0.3.61/js_sys/struct.Uint32Array.html.

**Comment by MatanelGordon** [comment]
Hy, thanks for responding.  I Found the problem:

I tried compiling the following code: 

```rust
#[wasm_bindgen]
pub fn test() ->  Uint32Array{
    let y = Uint32Array::from(vec![1,2,3,4].as_slice());
    return y;
}
```

The compilation was a success, yet I received no autocomplete for the "from" method nor any other method in my `CLion` IDE. 
Why could this happen? 

As it turns out, `Intelij Rust Plugin` does not know how to approach the way `js-sys` defined the array types (Uint8Array, Uint16Array, etc.) - since they used some advanced macros. 

However, when switched to `Vscode` I could get perfectly clear autocomplete as well as a brief explanation for each method.

So bottom line? Use `Vscode` when using wasm-bindgen.
